apps/server/src/modules/tool/external-tool/uc/external-tool.uc.spec.ts

```diff
@@ -334,6 +345,27 @@ describe(ExternalToolUc.name, () => {
 				);
 			});
 		});
+
+		describe('when external tool with lti11 config is given', () => {
```

Comment:
you should test if the tool was saved with the encrypted secret too

Reply:
I can test if the method was called with the encrypted secret, but I do not need a test for the returned object because we mock the return value in the setup...